interactive_engine/executor/ir/core/src/plan/ffi.rs

```diff
@@ -800,6 +848,29 @@ fn set_predicate(ptr: *const c_void, cstr_predicate: *const c_char, opt: InnerOp
     }
 }
 
+/// To set an operator's predicate from a pb predicate pointer
+fn set_predicate_with_pb(ptr: *const c_void, ptr_predicate_pb: FfiPbPointer, opt: InnerOpt) -> FfiResult {
```

Comment:
This is `set_predicate_pb`, rather than `set_predicate_with_pb`. Same apply to a few following functions. Also comment this function: `In the following functions, we can set expression directly from a pb pointer, which is parsed by the compiler instead of ffi function. `

Reply:
Done.